perceval/backends/github.py

```diff
@@ -68,6 +69,22 @@ def __init__(self, owner=None, repository=None, token=None,
         self.owner = owner
         self.repository = repository
         self.client = GitHubClient(owner, repository, token, base_url)
+        self._users = {}  # internal users cache
+
+    def _add_users(self, issue):
```

Comment:
Do not modify the `issue` object inside the method. For this case, I prefer to fetch the user inside the method and return it. Outside, you can assign it to `assignee_key`.